### DISCUSSION TITLE
How to get started with Octernship as I'm new to open source.

### DISCUSSION THREAD

**Discussion by Kishan1835** [community/community#50641] [GitHub Education] [answered]
I have joined the Appwrite Octernship and I don't know how to proceed forward

**Comment by queenofcorgis** [comment, marked as answer]
Welcome to the GitHub Community, @Kishan1835, we're happy you're here! You are more likely to get a useful response if you are posting your questions in the applicable category and are explicit about what your question entails--giving a few more details might help someone give you a nudge in the right direction. I've gone ahead and moved it for you. Good luck!
